docs/ispc.rst

```diff
@@ -902,6 +902,9 @@ preprocessor runs:
   * - ISPC_UINT_IS_DEFINED
     - 1
     - The macro is defined if uint8/uint16/uint32/uint64 types are defined in the ``ispc`` (it's defined in 1.13.0 and later)
+  * - ISPC_FP16_SUPPORTED
+    - 1
+    - The macro is defined if float16 type is supported by the target
```

Comment:
Let's be explicit about the semantics - this macros indicates if the type is fully supported on the target, whether through native hardware support or software emulation.